lib/internal/options.js

```diff
@@ -57,5 +61,6 @@ module.exports = {
   },
   getOptionValue,
   getAllowUnauthorized,
-  shouldNotRegisterESMLoader
+  shouldNotRegisterESMLoader,
+  noGlobalSearchPaths
```

Comment:
nit: ASCII order + trailing comma
```suggestion
  noGlobalSearchPaths,
  shouldNotRegisterESMLoader,
```